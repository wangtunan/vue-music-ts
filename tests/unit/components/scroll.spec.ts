import { shallowMount, Wrapper } from '@vue/test-utils'
import Scroll from '@/components/scroll/index.vue'

describe('scroll.vue', () => {
  let wrapper: Wrapper<Vue>
  beforeEach(() => {
    wrapper = shallowMount(Scroll,{
      slots: {
        default: `<div class="scroll-slot"></div>`
      }
    })
  })
  it('match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
  it('match snapshot', () => {
    expect(wrapper.findAll('.scroll-slot').exists()).toBe(true)
  })
})