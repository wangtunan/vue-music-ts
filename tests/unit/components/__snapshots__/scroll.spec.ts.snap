// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`scroll.vue match snapshot 1`] = `
<div>
  <div class="scroll-slot"></div>
</div>
`;
